Header header

uint8 bamper
int32 encoder1
int32 encoder2
int32 encoder3
int32 encoder4
int16 pwm1
int16 pwm2
int16 pwm3
int16 pwm4
